phase03/Model/Result.cs

```diff
@@ -0,0 +1,8 @@
+public class Result
```

Comment:
why instead of using constructor we set all the properties

Reply:
This part has changed complitetly, methods moved to separated classes and all are called in a loop iterating ISearcher list